src/sass/components/_main.scss

```diff
@@ -0,0 +1,120 @@
+body {
+  background-color: rgb(214, 208, 208);
```

Comment:
używaj koloru ze zmiennych proszę, jak nie ma zmiennej z potrzebnym kolorem to śmiało dodawaj